testsuite/classlibrary/TestDocument.sc

```diff
@@ -3,6 +3,40 @@ TestDocument : UnitTest {
 	// need a public place to communicate the result
 	classvar <>success;
 
+	test_save_method_writes_file {
+		var doc, file, str, path;
+		if(Platform.ideName == "scqt") {
+			path = Platform.defaultTempDir +/+ "test_document_save.txt";
+			if(File.exists(path)) {
+				this.assert(false, "Document save test file path should not exist: '%'".format(path));
+			} {
+				doc = Document(string: "abcde");
+				// note: AFAIK there is no signal coming from IDE to sclang
+				// to tell it that the document is ready in the editor.
+				// Without a signal, there is no Condition to wait on
+				// so we have no choice but to use a timed wait
+				0.1.wait;
+				doc.save(path);
+				0.1.wait;
```

Comment:
is this second wait necessary?

Reply:
BTW these waits are okay, i.e. there's no way to avoid them, AFAIU (due to async communication sclang <-> IDE).
Also, these tests won't be run in the automated test suite in GitHub Actions for the time being, since we don't run them in the IDE (just in sclang alone) and thus `Document` tests are disabled.